vars/deployViaGitops.groovy

```diff
@@ -342,6 +343,27 @@ protected String createBuildDescription(String pushedChanges) {
     return description
 }
 
+private printWelcomeAndConfigs(Map gitopsConfig){
+
+    print("""
```

Comment:
Fancy ASCII art :partying_face: 

Did you try if that looks good in jenkins?
With increase zoom level here on GitHub the lines are shifted and unreadable. 
I'm not sure, maybe using plain text would be less risky?
![image](https://github.com/user-attachments/assets/2e98b8e9-4caf-41a7-9312-fa0ecd5c8f05)